docs/manual/docs/cmd/spfx/project/project-upgrade.md

```diff
@@ -28,7 +28,7 @@ The `spfx project upgrade` command helps you upgrade your SharePoint Framework p
 
 This command doesn't change your project files. Instead, it gives you a report with all steps necessary to upgrade your project to the specified version of the SharePoint Framework. Changing project files is error-prone, especially when it comes to updating your solution's code. This is why at this moment, this command produces a report that you can use yourself to perform the necessary updates and verify that everything is working as expected.
 
-Using this command you can upgrade SharePoint Framework projects built using versions: 1.0.0, 1.0.1, 1.0.2, 1.1.0, 1.1.1, 1.1.3, 1.2.0, 1.3.0, 1.3.1, 1.3.2, 1.3.4, 1.4.0, 1.4.1, 1.5.0, 1.5.1, 1.6.0, 1.7.0, 1.7.1, 1.8.0 and 1.8.1.
+Using this command you can upgrade SharePoint Framework projects built using versions: 1.0.0, 1.0.1, 1.0.2, 1.1.0, 1.1.1, 1.1.3, 1.2.0, 1.3.0, 1.3.1, 1.3.2, 1.3.4, 1.4.0, 1.4.1, 1.5.0, 1.5.1, 1.6.0, 1.7.0, 1.7.1, 1.8.0, 1.8.1 and 1.9.1.
```

Comment:
I think that was about upgrading old version to latest where the latest is not really present because we cannot upgrade project build using 1.9.1. So if my assumption is right we should have `1.8.2` instead of `1.9.1`. It is still a miss tho, just different version number.

@waldekmastykarz is it like that?

Reply:
No. It is already in there. ;)
When I validate the current schema of a 1.9.1 on an existing project It says you are already on the right version.
Which if this is not supported an error will be raised about you cannot downgrade or something like that.

Reply:
There are two numbers in docs:

![image](https://user-images.githubusercontent.com/11164679/63689633-c44b2d00-c80a-11e9-96e1-03ca11202838.png)

![image](https://user-images.githubusercontent.com/11164679/63689648-cc0ad180-c80a-11e9-99ee-38197fab5ec8.png)

We should update both places. First, to v1.9.1 and the second to v1.8.2.

Reply:
Updated.

Reply:
Thank you Stefan! We'll merge it shortly.